src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,118 @@
+
+main {
+  display: flex;
+  justify-content: center;
+  align-content: stretch;
+}
+
+.catalog {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  flex-wrap: wrap;
+  width: (4 * $width-card) + (3 * $gap-card);
```

Comment:
For better readability, use variables for total number of cards and gaps, and then use them in calculations, e.g., $total-cards: 4; $total-gaps: 3;